playbooks/files/rax-maas/plugins/memcached_status.py

```diff
@@ -27,7 +27,7 @@
 
 
 VERSION_RE = re.compile('STAT version (\d+\.\d+\.\d+)(?![-+0-9\\.])')
-VERSIONS = ['1.4.14 (Ubuntu)', '1.4.15', '1.4.25 Ubuntu']
+VERSIONS = ['1.4.14 (Ubuntu)', '1.4.15', '1.4.25 Ubuntu', '1.4.39']
```

Comment:
I wonder if this is still valid? #574 has these options removed to support Rocky, so we might be able to do the same with RPCR work.

Reply:
Yes,  we'll test it out on an osp13 instance and see if it works. For now, we'll keep just the as it is.